include/libsemigroups/aho-corasick.tpp

```diff
@@ -97,8 +97,8 @@ namespace libsemigroups {
   }
 
   template <typename Iterator>
-  [[nodiscard]] AhoCorasick::index_type
-  AhoCorasick::traverse_trie(Iterator first, Iterator last) const {
+  AhoCorasick::index_type AhoCorasick::traverse_trie(Iterator first,
```

Comment:
What are the reasons for removing `[[nodiscard]]` here?

Reply:
Yeah, it should only be used with the function declaration in the hpp file